frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -19,16 +19,45 @@ export const PipelineRunDetails: React.FC<PipelineRunDetailsProps> = ({ obj: pip
           <ResourceSummary resource={pipelineRun} />
         </div>
         <div className="col-sm-6">
+          <SectionHeading text="Pipeline" />
           <dl>
-            <dt>Pipeline</dt>
             <dd>
-              <ResourceLink
-                kind={referenceForModel(PipelineModel)}
-                name={pipelineRun.spec.pipelineRef.name}
-                namespace={pipelineRun.metadata.namespace}
-              />
+              <React.Fragment key={pipelineRun.spec.pipelineRef.name}>
+                <dt>Name: {pipelineRun.spec.pipelineRef.name}</dt>
+                <ResourceLink
+                  kind={referenceForModel(PipelineModel)}
+                  name={pipelineRun.spec.pipelineRef.name}
+                  namespace={pipelineRun.metadata.namespace}
+                  inline
+                />
+              </React.Fragment>
             </dd>
           </dl>
+          <br />
+          {pipelineRun?.spec?.resources.length && (
+            <>
+              <SectionHeading text="Pipeline Resources" />
+              <dl>
+                {pipelineRun?.spec?.resources.map((res) => {
```

Comment:
Don't need to make subsequent checks check for null/undefined.

```suggestion
                {pipelineRun.spec.resources.map((res) => {
```